charts/k8s-monitoring/docs/examples/features/integrations/loki/values.yaml

```diff
@@ -0,0 +1,29 @@
+---
+cluster:
+  name: loki-integration-cluster
+
+destinations:
+  - name: prometheus
+    type: prometheus
+    url: http://prometheus.prometheus.svc:9090/api/v1/write
+  - name: loki
+    type: loki
+    url: http://loki.loki.svc:3100/api/push
+
+integrations:
+  loki:
+    instances:
+      - name: loki
+        logs:
+          enabled: true
+          labelSelectors:
+            app.kubernetes.io/name: loki
```

Comment:
This is outside of the `logs` field, isn't it?

Reply:
Fixed